DashAI/back/models/scikit_learn/bow_text_classification_model.py

```diff
@@ -22,7 +22,7 @@ class BagOfWordsTextClassificationModelSchema(BaseSchema):
 
     tabular_classifier: schema_field(
         component_field(parent="TabularClassificationModel"),
-        placeholder=100,
+        placeholder={"component": "SVC", "params": {}},
```

Comment:
At this moment, because of the dependency injection issue, it is not verifying if the component really exists or not, right?
What would happen if the component is not found? What if it has no schema?